### ISSUE TITLE
Show number of nodes in the network 

### ISSUE THREAD

**Issue by jjeffryes** [OpenBazaar/OpenBazaar-Server#264]
Copied from https://github.com/OpenBazaar/OpenBazaar-Client/issues/963 by @drwasho 

We can estimate this data from the seed server, and it's a nice non-essential feature that makes the user feel they're part of something big. Not sure what technical requirements are necessary for that.

**Comment by cpacia** [comment]
You can show the number of nodes that are connected. We can't really estimate total nodes.

**Comment by hoffmabc** [comment]
Number of nodes you're connected to is probably useful but don't sell it as size of network. It's untrue.

**Comment by BazaarBay** [comment]
Afaik it's possible to pretty accurately estimate network size using DHT data. Even with very limited view of network (neighbours alone). How buckets are filled and the keys are distributed, actually vies you pretty good estimate of network size.

**Comment by cpacia** [comment]
Yeah there's probably an algorithm that can be used to deduce the number of nodes from the makeup of the routing table. If someone wants to write that algo we could incorporate it.

**Comment by drwasho** [comment]
Want to take a stab at it @bazaarbay?

**Comment by drwasho** [comment]
Or @gubatron?

**Comment by BazaarBay** [comment]
@drwasho Sorry. As interesting task as it would be, I have to decline. I've got several bullet points (understatement) on my BB task list to get done.
